terraform/prod_cluster/eks-cluster.tf

```diff
@@ -0,0 +1,48 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.0.4"
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.24"
+
+  vpc_id                         = var.vpc_id
+  subnet_ids                     = var.subnet_ids
+  cluster_endpoint_public_access = true
+  cluster_security_group_id      = var.prod_security_group_id
+  aws_auth_users = [
+    {
+      userarn  = "arn:aws:iam::364159549467:user/liam"
+      username = "liam"
+      groups   = ["system:masters"]
+    },
+    {
+      userarn  = "arn:aws:iam::364159549467:user/cecilia"
+      username = "cecilia"
+      groups   = ["system:masters"]
+    },
+    {
+      userarn  = "arn:aws:iam::364159549467:user/natalie"
+      username = "natalie"
+      groups   = ["system:masters"]
+    }
+  ]
```

Comment:
Note that we should add removing us from this list to the Offboarding Jira epic - not urgent since the person's Amazon account would be shut down on departure, but a nice cleanliness thing.